src/Symfony/Component/HttpKernel/Attribute/LogLevel.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+/**
+ * @author Dejan Angelov <angelovdejan@protonmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class LogLevel
+{
+    /**
+     * @param \Psr\Log\LogLevel::* $level
```

Comment:
Can we use a `use` statement here to simplify the code?

```suggestion
     * @param LogLevel::* $level
```

Reply:
We cannot simplify it to just `LogLevel` because the attribute class has the same name, so we'll have to add an alias, thus I don't see any benefit from adding the `use` statement.

Reply:
This naming conflict is indeed unfortunate because the one thing you want to pass to the constructor of the `LogLevel` attribute is a `LogLevel` constant. This makes the everyday use of the attribute a bit cumbersome. Maybe we should rename the attribute?

Reply:
I'm open to suggestions as I couldn't think of a better name for the attribute.

Reply:
Dunno… `AsLogLevel`. `WithLogLevel`, `AssignLogLevel`, … 🤷🏻‍♂️

Reply:
MapLogLevel?
We already have a few Map* attributes
Worth renaming HttpStatus to MapHttpStatus?
Another idea could be At*

Reply:
Not sure about `Map` as it's used for argument mapping/binding/resolution AFAIK, and `At` feels weird for the status code. Another proposal: `#[ToLogLevel]` & `#[ToHttpStatus]`

Reply:
To me, it makes most sense to call it `WithLogLevel` (and `WithHttpStatus`).

Reply:
I renamed the class to `WithLogLevel` as it makes most sense to me. I can change it to something else if needed. I will also submit a PR to update the `HttpStatus` attribute accordingly after this is merged.